### PR TITLE
Update COMMITTERS.md

### PR DESCRIPTION
**Pull request by anaxceron**
updates Zowe Explorer squad list by updating membership and describing responsibilities for roles

### CODE REVIEW

**Review by adam-wolfe — CHANGES_REQUESTED** [review]
We also want Andrew in this list.

**Review by adam-wolfe — APPROVED** [review]

File: COMMITTERS.md [hunk 1]
```diff
@@ -54,22 +54,21 @@ Our definition of Lead, Committers and Contributors roughly follows https://gith
...
-| Adam Wolfe | @adam-wolfe | adam.wolfe@broadcom.com | Product Owner, Developer |
-| Na Tian (Tina) | @tiantn | tiantn@cn.ibm.com | Developer |
-| Qi Liang | @std4lqi | liangqi@cn.ibm.com | Developer |
-| Kutlu Ozel | @KutluOzel-b | kutlu.ozel@broadcom.com | Developer |
-| Peter Haumer | @phaumer | phaumer@us.ibm.com | Developer |
-| Anokhi Kastia | @anokhikastiaIBM | anokhi.kastia@ibm.com | Developer |
-| Lauren Li | @lauren-li | lauren.li@ibm.com | Developer |
+| Name | GitHub ID | Email | Roles | Responsibilities |
+|-|-|-|-|-|
+| Billie Jean Simmons | @JillieBeanSim | BillieJean.Simmons@ibm.com | Co-Lead, TSC Rep, Developer | Squad co-leader, builds product software |
```
Comment (JillieBeanSim) [inline COMMITTERS.md:59]: Thanks for putting this together @anaxceron 
While updating this file can we change the TSC Rep to be Timothy? Thanks

**Review by JillieBeanSim — CHANGES_REQUESTED** [review]

File: COMMITTERS.md (hunk 1 above)
Comment (anaxceron) [reply COMMITTERS.md]: Done!

**Comment by anaxceron** [comment]
@JillieBeanSim @zFernand0 @adam-wolfe -- please review

File: COMMITTERS.md [hunk 2]
```diff
@@ -56,20 +56,19 @@ Our definition of Lead, Committers and Contributors roughly follows https://gith
...
 | Fernando Rijo Cedeno | @zFernand0 | fernando.rijocedeno@broadcom.com | Co-Lead, Developer |
 | Jessica Tonda | @JTonda | jessica.tonda@broadcom.com | Scrum Master |
-| Ana Ceron | @anaxceron | ana.ceron@broadcom.com | Tech Writer |
+| Ana X. Ceron | @anaxceron | ana.ceron@broadcom.com | Tech Writer |
 | Rudy Flores | @rudyflores | rudyflores@ibm.com | Developer |
 | Trae Yelovich | @traeok | trae.yelovich@broadcom.com | Developer |
-| Timothy Johnson | @t1m0thyj | timothy.johnson@broadcom.com | Developer |
-| Adam Wolfe | @adam-wolfe | adam.wolfe@broadcom.com | Product Owner, Developer |
+| Timothy Johnson | @t1m0thyj | timothy.johnson@broadcom.com | TSC Rep, Developer |
+| Adam Wolfe | @adam-wolfe | adam.wolfe@broadcom.com | Security Squad Rep, Developer |
```
Comment (adam-wolfe) [inline COMMITTERS.md:66]: Is @zFernand0 the Security Squad rep for Zowe Explorer?

Comment (anaxceron) [reply COMMITTERS.md]: Fixed -- Fernando is sec rep for ZE; you are sec rep for CLI

**Review by adam-wolfe — APPROVED** [review]

**Review by JillieBeanSim — APPROVED** [review]

**Pull request merged**